### PR TITLE
README update for 1.7.0

### PR DESCRIPTION
**Pull request by baoy1**
# Description
README update for 1.7.0

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] NA

### CODE REVIEW

**Comment by baoy1** [comment]
e2e pass

**Review by forrestxia — APPROVED** [review]

**Comment by baoy1** [comment]
e2e pass

**Review by doriac11 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -86,6 +91,12 @@ The Terraform Provider for Dell PowerFlex is released and licensed under the MPL
...
   * [System](docs/resources/system.md)
+  * [Peer System](docs/resources/peer_system.md)
+  * [Replication Consistency Group](docs/resources/replication_consistency_group.md)
+  * [Replication Consistency Group Action](docs/resources/replication_consistency_group_action.md)
+  * [Replication Pair](docs/resources/replication_pair.md)
+  * [NVMe Host](docs/resources/nvme_host.md)
+  * [NVMe Target](docs/resources/nvme_target.md)
 
 ## List of Modules in Terraform Provider for Dell PowerFlex
   * [User](https://registry.terraform.io/modules/dell/modules/powerflex/latest/submodules/user)
```
Comment (doriac11) [inline README.md:102]: We should add Azure and AWS here: https://github.com/dell/terraform-powerflex-modules/pull/16 AWS was raised yesterday.

Comment (forrestxia) [reply README.md:102]: > We should add Azure and AWS here: [dell/terraform-powerflex-modules#16](https://github.com/dell/terraform-powerflex-modules/pull/16) AWS was raised yesterday.

@doriac11 when the aws pr can be merged?

Comment (doriac11) [reply README.md:102]: We will work to get AWS merged today @baoy1 @forrestxia. 
Plase add it as a link in the modules README

Comment (baoy1) [reply README.md:102]: Sure, @doriac11 ,  what is the module link for AWS?

Comment (doriac11) [reply README.md:102]: https://registry.terraform.io/modules/dell/modules/powerflex/latest/submodules/aws_install

**Comment by baoy1** [comment]
e2e pass

**Comment by baoy1** [comment]
e2e pass

**Pull request merged**